src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2699,22 +2700,63 @@ Mavlink::set_boot_complete()
 
 static void usage()
 {
-	PX4_INFO("usage: mavlink {start|status|stream|stop-all|boot_complete}");
-	PX4_INFO("    [-d device]");
+
+	PRINT_MODULE_DESCRIPTION(
+		R"DESCR_STR(
```

Comment:
It's removed on NuttX: https://github.com/PX4/Firmware/pull/7145/commits/62d4cb63e0c443851164a85e1eeab9d5eb69936e#diff-1d74d6cd3d156b55d40085f178a49cfeR356